### PR TITLE
[SFTY-925] Add Safety Center documentation

### PR DESCRIPTION
**Pull request by yaro-dd**
### What does this PR do? What is the motivation?

Safety Center is currently in public beta. This PR adds official documentation before we start promoting Safety Center more actively.

https://datadoghq.atlassian.net/browse/SFTY-925

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

Safety Center can be found both in [staging](https://dd.datad0g.com/organization-settings/safety-center) in [prod](https://app.datadoghq.com/organization-settings/safety-center).

### CODE REVIEW

**Review by daisukixci — APPROVED** [review]

**Comment by rtrieu** [comment]
Created DOCS-8300 for docs team review.

File: content/en/account_management/safety_center.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+### API Keys
+Unused **API keys** can facilitate unauthorized access to your organization if they become exposed on the internet. Unused keys need to be reviewed and revoked if your organization's infrastructure does not rely on them.
+
+{{< img src="account_management/safety_center/revoke-unused-api-key.png" alt="Revoking unused API key" style="width:70%;" >}}
+
+## Users
+In order to keep your organization safe it is important to follow best practices for user management. The **Users** page in **Safety Center** surfaces user-related security recommendations:
+
+- [User invites][7] that have not been accepted for 30+ days.
+- [Admin users][1] in case their number exceeds 10% of all users within an organization.
```
Comment (aliciascott) [inline content/en/account_management/safety_center.md:53]:
```suggestion
- [Admin users][1] in the event their number exceeds 10% of all users within an organization.
```

File: content/en/account_management/safety_center.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+### Pending Invites
+Having inactive user accounts or **stale pending user invites** increases the surface for a potential account takeover attack. That can be especially dangerous if inactive user accounts have high-privilege access. To keep the number of inactive users to a minimum consider either resending **old pending invites** or deleting them if those users do not need access to the Datadog platform.
+
+{{< img src="account_management/safety_center/resend-pending-invite.png" alt="Resending pending invite" style="width:70%;" >}}
+
+{{< img src="account_management/safety_center/delete-pending-invite.png" alt="Deleting pending invite" style="width:70%;" >}}
+
+### Admins
+Giving **admin access** to users without careful consideration increases potential security risks in case a user account with elevated privileges gets compromised. To keep the number of users with **admin access** low, review your admin users regularly and revoke admin privileges if users do not require them.
```
Comment (aliciascott) [inline content/en/account_management/safety_center.md:63]:
```suggestion
Giving **admin access** to users without careful consideration increases potential security risks in the event where a user account with elevated privileges gets compromised. To keep the number of users with **admin access** low, review your admin users regularly and revoke admin privileges if users do not require them.
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @yaro-dd this looks great, just a couple of small suggestions and please remove the updates to the non English language menu files, we only need to update the English files as we have a translation process that will update the others :)
Go ahead and request a re-review once the changes are in place, thanks !

**Comment by yaro-dd** [comment]
Thank you so much for the review @aliciascott! Addressed the comments ✅

**Review by aliciascott — APPROVED** [review]
Hi @yaro-dd sorry I had one more suggestion, since Safety Center is in public beta, we should add the banner at the top of the page:

It can be something simple like this:

<div class="alert alert-info">Datadog Safety Center is in public beta.</div>

Reference: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Callouts-(formerly-beta-callout)

If you don't mind adding that then I can go ahead and get this merged :)

**Pull request merged**